docs/source/changelog.rst

```diff
@@ -4,6 +4,46 @@
 Changes in nbconvert
 ====================
 
+6.0.0
+-----
+
+The following authors and reviewers contributed the changes for this release -- Thanks you all!
+
+- Philip Austin
+- Maarten Breddels
+- David Brochart
+- Matthias Bussonnier
+- Sylvain Corlay
+- Matthias Geier
+- Tyler Makaro
+- Matthew Seal
+- Jeremy Tuloup
+
+Significant changes
+~~~~~~~~~~~~~~~~~~~~
+
+Nbconvert 6.0 is a major release of nbconvert which includes many significant changes.
+
+- Unlike previous versions, nbconvert 6.0 relies on the [nbclient](https://github.com/jupyter/nbclient) package for the execute preprocessor, which allows for asynchronous kernel requests.
+
+- Nbconvert 6.0 includes a new "webpdf" exporter, which renders notebooks in pdf format through a headless web browser, so that complex outputs such as HTML tables, or even widgets are rendered in the same way as with the HTML exporter and a web browser.
+
+- The default template applied when exporting to HTML now produces the same DOM structure as JupyterLab, and is styled using JupyterLab's CSS. The pygments in use theme mimics JupyterLab's codemirror mode with the same CSS variables, so that custom jupyterlab themes could be applied. The classic notebook styling can still be enabled with
```

Comment:
```suggestion
- The default template applied when exporting to HTML now produces the same DOM structure as JupyterLab, and is styled using JupyterLab's CSS. The pygments in use theme mimics JupyterLab's codemirror mode with the same CSS variables, so that custom JupyterLab themes could be applied. The classic notebook styling can still be enabled with
```

Reply:
I don't understand: "The pygments in use theme mimics..."